src/transformers/modeling_utils.py

```diff
@@ -2344,15 +2345,37 @@ def from_pretrained(
         else:
             model_kwargs = kwargs
 
+        has_bnb_quantization_config = False
+        has_gptq_quantization_config = False
+        if hasattr(config, "quantization_config"):
+            # backward compatibility
+            has_bnb_quantization_config = (
+                hasattr(config.quantization_config, "load_in_8bit")
+                or config.quantization_config.get("quant_method", None) == "bitsandbytes"
+            )
+            has_gptq_quantization_config = config.quantization_config.get("quant_method", None) == "gptq"
+
+        gtpq_quantizer = None
```

Comment:
Let's call this a `quantizer` as there might be other methods in the future.

Reply:
sounds good